packages/introspect/src/e2e.test.ts

```diff
@@ -81,4 +99,74 @@ describe('e2e', () => {
                 }),
         });
     }, 100_000);
+
+    test('Add to Pagila', async () => {
+        await deleteData();
```

Comment:
it's generally safer to to the cleanup in a try/finally block, something like this:

```
try {
   await createTable()
} finally {
   await dropTable();
}
```